site/content/blogs/release-v121.en.md

```diff
@@ -27,6 +27,38 @@ Copilot v1.21 brings several new features and improvements:
 
 ## CloudFront Integration
 
+One of our first major additions to the Copilot environment manifest! CloudFront is an AWS Content Delivery Network which helps people deploy their applications across the globe, and now you can enable a distribution by simply setting `cdn: true` in your environment manifest and running `copilot env deploy`.
+
+### Currently supported features:
+- A distribution deployed in front of your Public Load Balancer
+- Public ingress restricted to the CloudFront distribution
+- HTTPS traffic through an imported certificate, or a Copilot-managed certificate
+
+### CloudFront with HTTPS
+Copilot makes this process easy! If you have an application initialized with a `--domain` specified during `app init`, the required certificate will be created for you. If you import your own certificates for your hosted zone, we'll walk you through importing the correct certificate for CloudFront.
+
+!!!info
+    CloudFront requires certificates to be in the `us-east-1` region. When importing a certificate, make sure to create your certificate in this region.
+
+First, create a certificate in the `us-east-1` region for your application with [AWS Certificate Manager](https://aws.amazon.com/certificate-manager/). You must add each domain associated with your application to this certificate. Once you've validated the certificate, you can add a field to your environment manifest to import the certificate for CloudFront:
+```yaml
+cdn:
+  certificate: arn:aws:acm:us-east-1:${AWS_ACCOUNT_ID}:certificate/13245665-h74x-4ore-jdnz-avs87dl11jd
+```
+Run `copilot env deploy`, then you can create an A-record in [Route 53](https://aws.amazon.com/route53/) which points to the CloudFront distribution created by Copilot. Just select to point the record to an `Alias` in the console, then Route 53 will show a valid DNS with any resources created in your account after you select the resource type (CloudFront in this case).
```

Comment:
If their environments are not in the same account as where their domains are hosted then this doesn't work. I think to be same we should suggest them to grab the CF distribution domain name and then use that as a value to create an A-record in the corresponding hosted zone.